utils/src/main/scala/com/salesforce/op/utils/spark/OpSparkListener.scala

```diff
@@ -197,7 +256,13 @@ case class StageMetrics private
   shuffleWriteTime: Long,
   shuffleBytesWritten: Long,
   shuffleRecordsWritten: Long
-) extends JsonLike
+) extends BaseStageMetrics with MetricJsonLike {
+  val duration: Option[Long] =
```

Comment:
is `duration` is not serialized?

Reply:
it was here in StageMetrics, but I forgot to add it to the Cumulative. Adding with tests